### DISCUSSION TITLE
Can't see the ultimate option after installing G-Helper

### DISCUSSION THREAD

**Discussion by Professor-01** [seerge/g-helper#631] [General]
I downloaded this software but still there's no ultimate mode in GPU modes.
I'm Using
System Model	ASUS TUF Gaming F15 FX506HCB_FX506HCZ
![Screenshot 2023-06-15 074255](https://github.com/seerge/g-helper/assets/107911341/67affb02-eafc-4a81-9766-91b975bf3489)

**Comment by seerge** [comment]
@Professor-01 most probably your device doesn't support that (i.e. doesn't have MUX)

**Reply by Professor-01** [reply]
Thankyou for your reply. 
But when i purchased this laptop it did come with that ultimate mode but suddenly after some updates in armoury crate it went away.

**Reply by seerge** [reply]
You are just not right
https://www.reddit.com/r/GamingLaptops/comments/13eejly/does_the_asus_tuf_f15_fx506hc_have_mux_switchif/
